garage/tf/algos/npo.py

```diff
@@ -308,12 +313,21 @@ def _build_policy_loss(self, i):
                         policy_dist_info_valid,
                         name="lr")
 
+                    ll = pol_dist.log_likelihood_sym(
+                        i.valid.action_var,
+                        policy_dist_info_valid,
+                        name="log_likelihood")
+
+                    vanilla = ll * adv_valid
```

Comment:
I think it might be more clear if you moved `vanilla` and `surr_vanilla` inside the `if` statement, even if it means copying a line or two.